sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -234,28 +234,30 @@ public DirectConnectionConfig setMaxRequestsPerConnection(int maxRequestsPerConn
     }
 
     /**
-     * Gets the request timeout interval
-     * This represents the timeout interval for requests
+     * Gets the network request timeout interval
+     * This represents the time to wait for response from network peer
      *
-     * Default value is 60 seconds
+     * Default value is 5 seconds
      *
-     * @return the request timeout interval
+     * @return the network request timeout interval
      */
-    Duration getRequestTimeout() {
-        return requestTimeout;
+    public Duration getNetworkRequestTimeout() {
+        return networkRequestTimeout;
     }
 
     /**
-     * Sets the request timeout interval
-     * This represents the timeout interval for requests
+     * Sets the network request timeout interval
+     * This represents the time to wait for response from network peer
      *
-     * Default value is 5 seconds
+     * Default value is 5 seconds.
+     * This value will impact the number of times retries happen within SDK.
```

Comment:
IMO this needs much more documentation - changing this value has impact on reliability/availability - and performance. This comment doesn't explain when/why a customer would want to change it and when changing it what risks they need to be aware of.

Reply:
Updated the comments.